src/resolve/resolved-conf.c

```diff
@@ -413,3 +413,47 @@ int manager_parse_config_file(Manager *m) {
 #endif
         return 0;
 }
+
+int config_parse_refuse_record_types(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+
+        if (isempty(rvalue)) {
+                m->refuse_record_types = set_free(m->refuse_record_types);
+                return 0;
```

Comment:
Not a matter, but for new conf parsers, please return 1 on success.
`return 0` means non-critical error in most cases.

Reply:
Yes. Just that i thought normal C would return 0 for success.

Reply:
`return 1` please.